tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "module": "CommonJS",
     "sourceMap": true,
     "strict": true,
-    "target": "es5",
+    "target": "es6",
```

Comment:
Why was this changed? :thinking:

Reply:
i changed that due to the JS static method .endsWith() not being valid with es5 as the target.